graphs.py

```diff
@@ -50,19 +52,43 @@ def render_graph(graph_data: dict):
 
     min_followers = min(graph_data['info_followers_per_day'])
     max_followers = max(graph_data['info_followers_per_day'])
-    print(max_followers)
+
     min_followers = floor(min_followers / 100) * 100  # round to bottom 100
     max_followers = ceil(max_followers / 100) * 100  # round to upper 100
-    max_tweets_likes = max(max(graph_data['info_tweets_per_day'], graph_data['info_likes_number']))
-    # (max(graph_data['info_tweets_per_day'], graph_data['info_likes_number'])  -> returns a list
-    max_tweets_likes = ceil(max_tweets_likes / 10) * 10 # round to upper 10
-    print(max_tweets_likes)
+    max_tweets_likes = max(graph_data['info_tweets_per_day'] + graph_data['info_likes_number'])
+
+    yaxis2 = dict(
+        title='tweets & likes',
+        showline=True,
+        zeroline=False,
+        titlefont=dict(
+            color='rgb(191, 63, 63)'
+        ),
+        tickfont=dict(
+            color='rgb(191, 63, 63)'
+        ),
+        overlaying='y',
+        side='right',
+
+    )
+
+    if max_tweets_likes < 20:  # use linear axis
```

Comment:
if the biggest point would be < 20, then it will use a linear scaling. Otherwise it will use logarithmic one